SwiftGen.playground/Pages/JSON-Demo.xcplaygroundpage/Contents.swift

```diff
@@ -31,12 +31,6 @@ import Foundation
 
 // swiftlint:disable identifier_name line_length number_separator type_body_length
 internal enum JSONFilesInline {
-  internal enum Configuration {
```

Comment:
I'm not sure this is correct? Also we need to add the right resources to playgrounds, @marcelofabri if you want I can do that part.

Reply:
it would be great if you could take over the playground piece 🙏